web/content/docs/benchmarks/reactive-transport/wetland/Wetland.pandoc

```diff
@@ -0,0 +1,123 @@
+The example model system is a treatment wetland.
+A treatment wetland is an engineered ecosystem that mimics natural occuring microbiological processes to treat wastewater.
+Basically, such a system consists of a basin filled with media (e.g. gravel or sand) and is passed through by wastewater.
+Multiple microbial organisms attached on the media surface and metabolize wastewater pollutants, which drives the cyclin of carbon and nutrients.
+
+
+This scenario is modified from a simulation run from Boog et al. (2019) (run *HMc_RTM_run88*), however, does not include heat transport.
+It models a horizontal subsurface flow  experimental system of 4.7 m in length, 1.2 m in width with a saturated depth of 0.9 m (Figure 1).
+The system was filled with gravel and loaded with domestic wastewater.
+The wastewater enters the system at a constant rate on the left side and leaves it via an overflow at the right side.
+
+
+The coupling of OGS-6 and IPhreeqc used for the simulation requires to include the transport of H^+^--ions to adjust the compulsory charge balance computated by Phreeqc.
+The solution by OGS-6--IPhreeqc will be compared to the solution by a coupling of OGS-5--IPhreeqc.
+
+
+## Problem Description
+
+The 1D model domain is 4.7 m long, discretized into 94 line elements and seperated in an influent, a treatment and an effluent zone (Figure 1).
+The domain is saturated at start--up ($p(t=0)=$ 8829 Pa).
+For the water mass influx a constant Neumann BC is set at the left side ($g_{N,\text{downstream}}^p$).
+For the water efflux, a constant pressure is defined at the right side boundary ($g_{D,\text{upstream}}^p$).
+
+![Model schematic and domain for the 1d simulation](../Wetland_domain.png)
+
+The microbiological processes will be modeled by a complex network of kinetic reactions according to the Constructed Wetland Model No. 1 (Langergraber, 2009).
+The network includes dissolved oxygen ($Do$) and nine different wastewater pollutants that are metabolized by six bacterial groups resulting in 21 kinetic reactions.
+A "clean" system is assumed at start-up, therefore, initial concentrations of wastewater pollutants and bacteria are set to 1.0e-4 and 1.0e-3 mg L^-1^, respectively.
+For wastewater pollutants entering the system, a time-dependent Dirichlet BC is defined at the influx point.
+Respective material properties, initial and boundary conditions are listed in Table 1--2.
+
+<!--![\n Network of microbiological reactions described by CWM1. Modified with permission from Samsó et al. (2013).](../Wetland_cwm1.png)-->
+
+
+-----------------------------------------
+
+|Parameter | Description | Value | Unit |
+|:-------- | :---------- | :---- | :--- |
+| Influent & effluent zone |||
+| $\phi$   | Porosity    | 0.38  | - |
+| $\kappa$ | Permeability | 1.0e-7 | m^2^ |
+| $S$   | Storage | 0.0     | - |
+| $a_L$ | long. Dispersion length | 0.45   | m |
```

Comment:
Why is the transversal dispersion length not mentioned?

Reply:
As it is a 1D model there is only longitudinal dispersion.